package.json

```diff
@@ -39,8 +39,5 @@
   "volta": {
     "node": "18.12.1",
     "yarn": "1.22.19"
-  },
-  "devDependencies": {
-    "@babel/plugin-proposal-private-property-in-object": "7.21.11"
   }
```

Comment:
while running test this was suggested 
![Screenshot from 2024-08-18 19-09-12](https://github.com/user-attachments/assets/c598ccf3-a6b4-4da7-9866-ffc7d42da6a6)

Reply:
Great catch @shubhdevelop! But even this has been deprecated. 

We need another one: https://www.npmjs.com/package/@babel/plugin-proposal-private-property-in-object

Reply:
Sure, I'll check for the compatibility, and try to figure if this fixes it

Reply:
[@babel/plugin-transform-private-property-in-object](https://babeljs.io/docs/babel-plugin-transform-private-property-in-object)

This package does fix the issue. 

![image](https://github.com/user-attachments/assets/66b29e93-28fa-47c5-9f5e-93ead556877e)

Reply:
Added in review changes commit e8845a384b469b723838ad795f04ddf744e2b084